templatizer.js

```diff
@@ -35,7 +37,16 @@ module.exports = function (templateDirectories, outputFile, options) {
     if (typeof templateDirectories === "string") {
         templateDirectories = [templateDirectories];
     }
-    
+
+    var amdModuleDependencies = '';
+    var separator = '';
+    _.each(options.amdDependencies, function(element) {
+        amdModuleDependencies = separator + '\'' + element + '\'';
```

Comment:
This looks like it's reassigning `amdModuleDependencies` every time through the loop so that it'll only have the last element. It should probably be `+=` right?